src/OrchardCore.Modules/OrchardCore.PublishLater/Models/PublishLaterPart.cs

```diff
@@ -1,8 +1,12 @@
+using Microsoft.AspNetCore.Mvc.ModelBinding;
 using OrchardCore.ContentManagement;
 
 namespace OrchardCore.PublishLater.Models;
 
 public class PublishLaterPart : ContentPart
 {
     public DateTime? ScheduledPublishUtc { get; set; }
+
+    [BindNever]
+    public bool DoingScheduledPublish { get; set; }
```

Comment:
I don't think this should be added

Reply:
When an item is published whether it's by a user or background process. The publish DateTime should be the current time when the event took place. If you schedule the publish at 3PM and it gets published 3:05PM, the published time should be 3:05PM not 3:00PM. Do you agree?

Reply:
I think the publish date should be the actual publish date not the scheduled date. Schedule date should be to schedule the e time to process the request and the publish data should be the exact time it was published